pom.xml

```diff
@@ -155,6 +155,7 @@
 
     <version.org.jboss.marshalling.api>1.2.3.GA</version.org.jboss.marshalling.api>
 
+    <version.jacoco.plugin>0.7.5.201505241946</version.jacoco.plugin>
```

Comment:
To use the same nameing convention you should add "org" after version

<version.org.jacoco.plugin>0.7.5.201505241946</version.org.jacoco.plugin>

Reply:
I followed the convention from kie-parent - and you are generally right, speaking about dependencies. Although if you take a look e.g. on surefire plugin version declaration (and few others below):
https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L182
for plugins, the full artifactId seems to be omitted.

I have no problem in changing it if you insist, but it would be good to follow the same conventions across the KIE group.

Reply:
@rsynek is right here look at jboss-parent https://github.com/jboss/jboss-parent-pom/blob/master/pom.xml#L83-L131

Reply:
@mareknovotny @rsynek in that case +1 for me :)